src/auth/jwt/jwt.strategy.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from "@nestjs/common";
+import { ConfigService } from "@nestjs/config";
+import { PassportStrategy } from "@nestjs/passport";
+import { ExtractJwt, Strategy } from "passport-jwt";
+
+import { type JwtUser } from "./jwt.interface";
+
+@Injectable()
+export class JwtStrategy extends PassportStrategy(Strategy) {
+  constructor(configService: ConfigService) {
+    const secrect: string = configService.get("appSecret") ?? "Cophr_appSecret";
```

Comment:
is it secure to provide a default secret? may be a potential security issue

Reply:
When the backend init (src/config/env.validation.ts) will validate JWT_SECRET is not empty, so make it match will return string or undefined.

[fix: delete default secret](https://github.com/Cophr/backend/pull/10/commits/b106b164bff3e9c9f9b9959b7866ffe685fc702c)